### PR TITLE
Convert to ES6 that is supported on Node 4, commonjs modules and remove Babel

### PR DESCRIPTION
**Pull request by corbinu**
Lets try this one more time

This removes Babel, converts to commonjs modules and removes other ES6 features not supported on Node 4. I think it was all destructuring.

Updated version of #8 which removes .defaults and rebased on master

### CODE REVIEW

File: src/referencer.js [hunk 1]
```diff
@@ -222,11 +227,13 @@ export default class Referencer extends esrecurse.Visitor {
             });
         }
 
-        // Skip BlockStatement to prevent creating BlockStatement scope.
-        if (node.body.type === Syntax.BlockStatement) {
-            this.visitChildren(node.body);
-        } else {
-            this.visit(node.body);
+        if (node.body) {
```
Comment (JamesHenry) [inline src/referencer.js]: This code change should not be in this PR

Comment (corbinu) [inline src/referencer.js]: Sorry thought I had gotten that out with all my reattempts one sec

**Comment by JamesHenry** [comment]
Please can you rebase this (and all your other PRs)? Merging in master is less safe than rebasing, and maintaining a messy commit history does not really make sense on small-scale PRs.

We are currently not enforcing it, but commits should be following our contributing guidelines, as with any other ESLint project.

**Comment by corbinu** [comment]
@JamesHenry sure one sec

**Comment by not-an-aardvark** [comment]
@corbinu In the future, can you update an existing PR rather than closing/recreating a new PR every time a change is requested? It makes things much easier to review when all the comments are in one place.

**Comment by corbinu** [comment]
@not-an-aardvark Yes I am sorry I was having trouble getting the rebase to work after mistakenly merging

**Pull request closed without merging**